app/frontend/translations.ts

```diff
@@ -130,6 +136,15 @@ const translations = {
   PoolRegNoTtl: () =>
     'TTL parameter is missing in the transaction. It is explicitly required even for the Allegra era.',
   PoolRegTxParserError: ({message}) => `Parser error: ${message}`,
+
+  Error: ({message}) => {
+    const errors = {
+      'NotFoundError: The device was disconnected.': `${message}${ledgerConnectError}`,
+      'AbortError: The transfer was cancelled.': `${message}${ledgerConnectError}`,
+      "SyntaxError: Failed to execute 'postMessage' on 'Window': Invalid target origin 'chrome-extension://kmendfapggjehodndflmmgagdbamhnfd' in a call to 'postMessage'": `${message}`,
```

Comment:
where does that error message come from? Not sure if that extension identifier is global

Reply:
I mentioned it briefly in the PR description, its pretty rare and regarding the extension identifier, I haven't checked it but I was researching it couple of times I always came to that one crypto-something extension in chrome that somehow causes this.

Reply:
I see, it's apparently the CryptoToken extension allowing 2-step verification, likely via U2F: https://askubuntu.com/questions/844090/what-is-cryptotokenextension-in-chromium-extensions
From sentry it seems that the users that had that issue so far were using Ledger but of course, not necessarily. Maybe I would reword the troubleshooting suggestion to "If you are using Ledger Nano, ..." and put it here as well

Reply:
and please clarify with a comment where that arbitrary string comes from and why it's here

Reply:
Thanks, I should have commented it. :+1: